model_bakery/baker.py

```diff
@@ -46,8 +57,11 @@ def _valid_quantity(quantity: Optional[Union[str, int]]) -> bool:
     return quantity is not None and (not isinstance(quantity, int) or quantity < 1)
 
 
+SpecificModelType = TypeVar("SpecificModelType", bound=ModelBase)
+
+
 def make(
-    _model: Union[str, Type[ModelBase]],
```

Comment:
I know this was existing code, but I believe this type is actually not correct. `ModelBase` is the _metaclass_ used by all "model" classes, however, the more appropriate type (in my naive opinion) should be `django.db.models.base.Model` (or in the form of an import: `from django.db.models import Model`), as that is the actual class from which models derive.

Therefore, there should be no need for the `TypeVar` if the correct `Type` annotation class were to be used.

Reply:
No, the `TypeVar` would still be necessary. If I remove it, the return type would be `Union[Model, List[Model]]`, and that wouldn't help, as this removes necessary type information needed for proper code completion of user-defined models. To allow deducing the correct user-defined model class in the return type, it needs to know the specific model type passed in.

I have changed the bound type from `ModelBase` to `Model` and rebased the branch onto the current main branch.

Reply:
Ah, I see. Yes you are correct. Now with the proper base I think this is good.